eng/tools/TypeSpecValidation/package.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "typespec-validation",
+    "version": "0.0.1",
+    "type": "module",
+    "main": "dist/TypeSpecValidation.js",
+    "bin": {
+        "tsv": "cmd/tsv.js"
+    },
+    "dependencies": {
+        "simple-git": "^3.16.0"
+    },
+    "devDependencies": {
+        "@types/node": "^18.16.18"
```

Comment:
We should be using `@types/node@^16`, but are blocked due to this bug:

https://github.com/DefinitelyTyped/DefinitelyTyped/discussions/64875